cognite/client/utils/_time.py

```diff
@@ -102,6 +106,26 @@ def ms_to_datetime(ms: int | float) -> datetime:
     return datetime(1970, 1, 1, tzinfo=timezone.utc) + timedelta(milliseconds=ms)
 
 
+def datetime_to_ms_iso_timestamp(dt: datetime) -> str:
+    """Converts a datetime object to a string representing a timestamp in the ISO-format expected by the Cognite GraphQL API.
+
+    Args:
+        dt (datetime): Naive or aware datetime object. Naive datetimes are interpreted as local time.
+
+    Raises:
+        TypeError: If dt is not a datetime object
+
+    Returns:
+        str: Timestamp string in ISO 8601 format with milliseconds
+    """
+    if isinstance(dt, datetime):
+        if _datetime_is_tz_unaware(dt):
```

Comment:
Imo you dont need this function

Reply:
I would agree if it was one simple check, but as we discussed earlier we need two which is not obvious :) And I think this could be useful for other functions as well. But I should have called it "naive" instead of "unaware". Agree?

Reply:
Agree